app/models/code_school.rb

```diff
@@ -1,5 +1,5 @@
 class CodeSchool < ApplicationRecord
   validates :name, :url, :logo, presence: true
   validates_inclusion_of :full_time, :hardware_included, :has_online, :online_only, :in => [true, false]
-  has_many :locations, dependent: :destroy
+  has_many :locations, -> { order('state ASC, city ASC') }, dependent: :destroy
```

Comment:
Bake this sort ordering as default into any query for a school's locations.

Reply:
How does this work and what does it do?